src/nucleus/jdg.ml

```diff
@@ -669,7 +673,7 @@ let congr_lambda ~loc (EqTy (ctxa, ta1, ta2))
                  (EqTy (ctxb, b1, b2))
                  (EqTerm (ctxe, e1, e2, ty_e)) =
   if not (TT.alpha_equal_ty b1 ty_e)
-  then error ~loc (RuleInputMismatch ("congr-lambda", b1, ty_e))
+  then error ~loc (RuleInputMismatch ("congr-lambda", b1, "???", ty_e, "???"))
```

Comment:
The left hand side body type, and the type at which the equality on the bodies is happening.
ie the caller said
```
|- A1 =[Type] A2
x : A1 |- B1 =[Type] B2
x : A1 |- e1 =[C] e2
_____________________________________________________
|- lambda x : A1. B1. e1 =[forall x : A1, B1] lambda x : A2. B2. e2
```
but `C` should be `B1`.